src/service/timer-impl.js

```diff
@@ -0,0 +1,161 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Requires polyfills in immediate side effect.
+import '../polyfills';
```

Comment:
looks like it was to fix something in closure compiler dependency ordering https://github.com/ampproject/amphtml/commit/084eac2f60e7e99da0b33f89a2517a0735686fe6